server.api/src/main/java/org/kaazing/gateway/server/impl/VersionUtils.java

```diff
@@ -238,4 +203,35 @@ private static boolean getProductInfoFromJar(String pathEntry, Map<String, Attri
         }
         return result;
     }
+
+    private static class GatewayProductVersionType {
```

Comment:
Rename this as ProductInfo. The class should have the members : title, version, edition, dependencies.